products/views.py

```diff
@@ -1,3 +1,46 @@
-from django.shortcuts import render
+import re
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q
+
+from .models          import Product
+
+class ProductOptionView(View):
+    def get(self, request):
+        option = request.GET.get('option', None)
+        offset = request.GET.get('offset', 1)
+        limit  = request.GET.get('limit', 12)
```

Comment:
`offset`, `limit` 잘 적용해주셨네요~ 👍👍👍 
```suggestion
        offset = int(request.GET.get('offset', 1))
        limit  = int(request.GET.get('limit', 12))
```

Reply:
감사합니다!